Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -117,7 +117,7 @@ def parse_args(args):
     option_group_definitions.append(("Feature Switches", [
         optparse.make_option("--complex-text", action="store_true", default=False,
             help="Use the complex text code path for all text (OS X and Windows only)"),
-        optparse.make_option("--accelerated-drawing", action="store_true", default=False,
+        optparse.make_option("--accelerated-drawing", action="store_true", default=True,
```

Comment:
This feels like a big change worthy of a separate PR (and no doubt, the reason for `webkitpy` test failures.

Reply:
Yeah this is going to be enabled here: https://bugs.webkit.org/show_bug.cgi?id=246285. I wanted to make sure these updates cause the tests to pass with this flag enabled. I'll remove it once I get the results.

Reply:
Makes sense!